Source/Core/DolphinQt2/Config/SettingsWindow.cpp

```diff
@@ -21,7 +21,7 @@ SettingsWindow::SettingsWindow(QWidget* parent) : QDialog(parent)
 {
   // Set Window Properties
   setWindowTitle(tr("Settings"));
-  resize(720, 600);
+  resize(500, 450);
```

Comment:
It makes it a bit smaller than what I was aiming for, but I guess that's better than magic numbers